docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_prefix ``cliquet``     | The prefix to use when sending data to statsd.                           |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_url: ``None``          | The URL to use to connect to the statsd host. e.g.                       |
+|                                       | ``udp://localhost:8125``                                                 |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Logging with Heka
+:::::::::::::::::
+
+Mozilla Services standard logging format can be enabled using:
+
+.. code-block:: ini
+
+    cliquet.logging_renderer = cliquet.logs.MozillaHekaRenderer
+
+
+With the following configuration, all logs are redirected to standard output
+(See `12factor app <http://12factor.net/logs>`_):
+
+.. code-block:: ini
+
+    [loggers]
+    keys = root
+
+    [handlers]
+    keys = console
+
+    [formatters]
+    keys = heka
+
+    [logger_root]
+    level = INFO
+    handlers = console
+    formatter = heka
+
+    [handler_console]
+    class = StreamHandler
+    args = (sys.stdout,)
+    level = NOTSET
+
+    [formatter_heka]
+    format = %(message)s
+
+
+Handling exceptions with Sentry
+:::::::::::::::::::::::::::::::
+
+Requires the ``raven`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+Sentry logging can be enabled, `as explained in official documentation
+<http://raven.readthedocs.org/en/latest/integrations/pyramid.html#logger-setup>`_.
+
+.. note::
+
+    The application sends an *INFO* message on startup, mainly for setup check.
+
+
+Monitoring with StatsD
+::::::::::::::::::::::
+
+Requires the ``statsd`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+StatsD metrics can be enabled (disabled by default):
+
+.. code-block:: ini
+
+    cliquet.statsd_url = udp://localhost:8125
+    # cliquet.statsd_prefix = cliquet.project_name
+
+
+Monitoring with New Relic
+:::::::::::::::::::::::::
+
+Requires the ``newrelic`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+Enable middlewares as described :ref:`here <configuration-middlewares>`.
+
+New-Relic can be enabled (disabled by default):
+
+.. code-block:: ini
+
+    cliquet.newrelic_config = /location/of/newrelic.ini
+    cliquet.newrelic_env = prod
+
+.. _configuration-authentication:
+
+Authentication
+==============
+
+By default, *Kinto* relies on *Basic Auth* to authenticate users.
+
+User registration is not necessary. A unique user idenfier will be created
+for each couple of ``username:password``.
+
+*Kinto* is compatible with *Firefox Account*.  Install and
```

Comment:
"Install and configure" is really brief - maybe add a bit more of a description here, so that it lines up with the rest of the documentation in terms of verbosity.